_translations/en.yml

```diff
@@ -367,11 +385,47 @@ en:
+      continue to be buried under every block after it, which will
+      exponentially consolidate this consensus and decrease the risk of
+      a reversed transaction.
+      Each confirmation takes between a few seconds and 90 minutes, with
+      10 minutes being the average. If the transaction pays too low a
+      fee or is otherwise atypical, getting the first confirmation can
+      take much longer.  Every user is free to determine at what
+      point they consider a transaction sufficiently confirmed, but <a href="#you-need-to-know##instant">6 confirmations</a>
+      is often considered to be as safe as waiting 6 months on a credit
+      card transaction.
+
     fee: "How much will the transaction fee be?"
-    feetxt1: "Most transactions can be processed without fees, but users are encouraged to pay a small voluntary fee for faster confirmation of their transactions and to remunerate miners. When fees are required, they generally don't exceed a few pennies in value. Your Bitcoin client will usually try to estimate an appropriate fee when required."
-    feetxt2: "Transaction fees are used as a protection against users sending transactions to overload the network. The precise manner in which fees work is still being developed and will change over time. Because the fee is not related to the amount of bitcoins being sent, it may seem extremely low (0.0005 BTC for a 1,000 BTC transfer) or unfairly high (0.004 BTC for a 0.02 BTC payment). The fee is defined by attributes such as data in transaction and transaction recurrence. For example, if you are receiving a large number of tiny amounts, then fees for sending will be higher. Such payments are comparable to paying a restaurant bill using only pennies. Spending small fractions of your bitcoins rapidly may also require a fee. If your activity follows the pattern of conventional transactions, the fees should remain very low."
+    feetxt1: >
+      Transactions can be processed without fees, but users are
+      encouraged to pay a voluntary fee for faster confirmation of
+      their transactions and to remunerate miners. When fees are
+      required, they currently don't exceed a few pennies in value,
+      but this may increase over time. Your
+      Bitcoin client will pay what it thinks is an appropriate fee.
+
+    feetxt2: >
+      Transaction fees are used as a protection against users sending
+      transactions to overload the network. The precise manner in which
+      fees work is still being developed and will change over time.
+      Because the fee is not related to the amount of bitcoins being
+      sent, it may seem extremely low or unfairly high. Instead, the
+      fee is relative to the number of bytes in the transaction, so
+      using multisig or spending multiple previously-received amounts
+      may cost more than simpler transactions. If your activity follows
+      the pattern of conventional transactions, you won't have to pay
+      unusually high fees.
```

Comment:
> the fee is defined by attributes such as the number of bytes in the transaction and
transaction recurrence.

I think this is misleading.  Miners tend to use Bitcoin Core's fee/priority evaluation code, which I believe works like this:

- Fees are evaluated purely based on fee-per-byte.  The age and value of the input (priority) plays no role here.

- If the transaction doesn't qualify for inclusion in the block based on its fee, its priority is evaluated.  Here the fee plays no additional role.

In other words, fee and priority are considered separately, but this sentence makes me feel like they're averaged together to allow high-priority transactions to pay lower fees for the same processing speed as lower-priority transactions.

> Such payments are comparable to paying a restaurant bill using only pennies.

I see where this analogy was going, but it breaks down for me.  I have personal experience refusing a transaction paid in an excessive number of pennies from when I worked at McDonalds in high school, 17 years ago.  I think that would be the usual response to that situation; I don't know of any business besides a bank that would charge more to process loose change.

> Spending small fractions of your bitcoins rapidly may also require a fee.

I think this could confuse people about how small "small fractions" are or how rapid "rapidly" is.  According to my node's `estimatepriority`, getting a free transaction confirmed within the next 25 blocks requires 331411429.8671501 priority.  So to send a 250-byte transaction with an input worth 0.1 BTC (10,000,000 satoshis) requires that input be 8286 blocks old: `331411429.8671501 / 10000000 * 250`

I wouldn't describe $30 USD worth of bitcoin as small or two months worth of waiting as rapid, so I think this sentence should be removed and that we should generally treat free transactions as something for experts and curious people only.

> transaction recurrence.

In case you revise, I'd suggest using a different term.  Recurrence is meant to be applied to similar events separated by time.  I get that receiving one Bitcoin transaction seems similar to receiving another Bitcoin transaction, but people tend to think of transactions based on what was exchanged---not the medium the exchange occurred through---so selling a pocket knife for 0.1 BTC in one transaction and a flashlight for 0.2 BTC in a different transaction doesn't seem like recurrence.

---

My suggestion would be to use the text in the commit, which does include examples, but I'm happy to continue working on revisions of this part of the paragraph.  (I'll be pushing updated text in a moment based on the rest of your commit comments.)